kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -208,11 +240,31 @@
         this.$emit('showLanguageModal');
         this.userMenuDropdownIsOpen = false;
       },
+      mapSyncStatusOptionToLearner() {
+        if (this.userSyncStatus) {
```

Comment:
Empty arrays are not falsey in Javascript, so I don't know when this would not be true - unless the API endpoint can return `null`?